src/cli/utils/flags.go

```diff
@@ -115,13 +116,28 @@ func AddRunModeFlag(cmd *cobra.Command, persistent bool) {
 	cobra.CheckErr(fs.MarkHidden(RunModeFN))
 }
 
-// AddUserFlag adds the --user flag.
+// AddUserFlag adds the --user and --mailbox flag.
 func AddUserFlag(cmd *cobra.Command) {
-	cmd.Flags().StringSliceVar(
+	flags := cmd.Flags()
+
+	flags.StringSliceVar(
 		&UserFV,
 		UserFN, nil,
 		"Backup a specific user's data; accepts '"+Wildcard+"' to select all users.")
-	cobra.CheckErr(cmd.MarkFlagRequired(UserFN))
+
+	cobra.CheckErr(flags.MarkDeprecated(UserFN, fmt.Sprintf("user flag is deprecated. Use - %s", MailBoxFN)))
```

Comment:
```suggestion
	cobra.CheckErr(flags.MarkDeprecated(UserFN, fmt.Sprintf("use --%s instead", MailBoxFN)))
```

IMO, this would be better. The  will make the whole thing to be `Flag --user has been deprecated, use --mailbox instead`.

Reply:
![updatedMsg](https://user-images.githubusercontent.com/23420056/231387877-a23a4691-0a34-4053-9352-29820c6d24ce.png)

Reply:
I was thinking we could mimic mailbox as a flag to avoid confusion, ie `--mailbox.